templates/python/cloudevents/pyproject.toml

```diff
@@ -0,0 +1,26 @@
+[project]
```

Comment:
this is for `poetry`, right? 

Would it make sense to have a `requirements.txt` as well - for pip? 
Or would this not work? 

I am not really an expert there, but would perhaps like in some section mentioned why `pyproject` / `poetry` vs `pip` ?

Reply:
Yes this fully works, and is supported by `pip`.  This method, using a pyproject.toml appears to be the widely supported current standard amongst build systems.  Also, while the `func-pyuthon` repository (middleware) uses poetry (so we can take advantage of some of its more advanced features) the template written out for Function authors, defined here, uses the more widely used (and less complex to setup) hatchling build system.  This can be swapped out by the Function author by editing the `build-system` section of the pyproject.toml.  IIUC this is pretty widely understood by python developers.